cmd/Makefile.am

```diff
@@ -471,13 +485,18 @@ system_shutdown_unit_tests_SOURCES = \
 system_shutdown_unit_tests_LDADD = libsnap-confine-private.a libsnap-confine-private-test-support.a
 system_shutdown_unit_tests_CFLAGS = $(AM_CFLAGS) $(GLIB_CFLAGS)
 system_shutdown_unit_tests_LDADD +=  $(GLIB_LIBS)
+system_shutdown_unit_tests_LDFLAGS =  $(system_shutdown_system_shutdown_LDFLAGS)
 endif
 
 ##
 ## snap-gdb-shim
 ##
 
+if !ENABLE_SANITIZE
```

Comment:
I suppose you do not add `SANITIZE_*` to `snap_gdb_shim_snap_gdb_shim_*`. So that would still build, right? Is it just to disable what we are not testing here?

Reply:
Bit of both. We don't test it and it won't build. Even with -static-libasan dlopen() is retained for some reason. I guess it just makes libasan parts to be linked statically, but still relies on hijacking libc's malloc dynamically.